lib/locales/de.js

```diff
@@ -11,7 +11,68 @@
+        'Double-click to zoom back out': 'Herauszoomen durch Doppelklick',                         // plots/cartesian/dragbox.js:299
+        'Download plot as a png': 'Download als png',                                // components/modebar/buttons.js:52
+        'Edit in Chart Studio': 'Im Chart Studio bearbeiten',                                  // components/modebar/buttons.js:76
+        'IE only supports svg.  Changing format to svg.': 'IE unterstützt nur SVG-Dateien.  Format wird zu SVG gewechselt.',        // components/modebar/buttons.js:60
+        'Lasso Select': 'Lassoauswahl',                                          // components/modebar/buttons.js:112
+        'Orbital rotation': 'Orbitalrotation',                                      // components/modebar/buttons.js:279
+        'Pan': 'Verschieben',                                                   // components/modebar/buttons.js:94
+        'Produced with Plotly': 'Erstellt mit Plotly',                                  // components/modebar/modebar.js:256
+        'Reset': 'Zurücksetzen',                                                 // components/modebar/buttons.js:432
+        'Reset axes': 'Achsen zurücksetzen',                                            // components/modebar/buttons.js:148
+        'Reset camera to default': 'Kamera auf Standard zurücksetzen',                               // components/modebar/buttons.js:314
+        'Reset camera to last save': 'Kamera auf letzte Speicherung zurücksetzen',                             // components/modebar/buttons.js:322
+        'Reset view': 'Ansicht zurücksetzen',                                           // components/modebar/buttons.js:583
+        'Reset views': 'Ansichten zurücksetzen',                                           // components/modebar/buttons.js:529
+        'Show closest data on hover': 'Zeige nähste Daten beim Überfahren',                            // components/modebar/buttons.js:157
+        'Snapshot succeeded': 'Snapshot erfolgreich',                                    // components/modebar/buttons.js:66
+        'Sorry, there was a problem downloading your snapshot!': 'Es gab ein Problem beim Herunterladen des Snapshots', // components/modebar/buttons.js:69
+        'Taking snapshot - this may take a few seconds': 'Erstelle einen Snapshot - dies kann einige Sekunden dauern',         // components/modebar/buttons.js:57
+        'Zoom': 'Zoom',                                                  // components/modebar/buttons.js:85
+        'Zoom in': 'Hineinzoomen',                                               // components/modebar/buttons.js:121
+        'Zoom out': 'Herauszoomen',                                              // components/modebar/buttons.js:130
+        'close: ': 'schließen',                                                // traces/ohlc/transform.js:139
```

Comment:
@alexcjohnson The English key (`'close: '`) has a space after the colon. Is this correct? In https://github.com/plotly/plotly.js/pull/2217#issuecomment-353177120, you said that "No translation keys or values should start or end with spaces"

Reply:
'Schluss: ' (https://books.google.com.au/books?id=UjgPCgAAQBAJ&lpg=PA93&dq=balkenchart&pg=PA93#v=onepage&q&f=false)

Reply:
Right, we should remove all of these trailing spaces. Since this is the first full translation @JonasGutermuth was in uncharted territory, and `translation-keys.txt` doesn't make this clear due to the path/line comments. My apologies for not making this more clear earlier!